staging/src/k8s.io/node-api/hack/generate-bindata.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
What is the purpose of the generated bindata?

Reply:
> Can we reuse https://github.com/kubernetes/kubernetes/blob/master/hack/generate-bindata.sh somehow?

Unfortunately, it's common practice to copy `hack/` scripts into related kubernetes repos. The relationship between staging & kubernetes/kubernetes is confusing, but my understanding is that it should be functional as an independent repo.

Reply:
> What is the purpose of the generated bindata?

 The bindata essentially packages raw data files into go code so that they're included as part of the go binary. This way, the CRD can be included simply by importing this package.